a2p_importpart.py

```diff
@@ -363,7 +363,7 @@ def updateImportedParts(doc):
                     importUpdateConstraintSubobjects( doc, obj, newObject )# do this before changing shape and mux
                     if hasattr(newObject, 'muxInfo'):
                         obj.muxInfo = newObject.muxInfo
-                    # save Placement becaause following newObject.Shape.copy() ist resetting it to zeroes...
+                    # save Placement because following newObject.Shape.copy() isn't resetting it to zeroes...
```

Comment:
`isn't` or `is` ?

Reply:
"is" is correct..
Thank you for the PR. I rebased it to devel branch